### PR TITLE
add coda, harp and muninn

### PR DESCRIPTION
**Pull request by srepmub**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by svniemeijer** [comment]
I confirm to be a maintainer

File: recipes/coda/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  host:
+    - python
+    - jpeg
+    - zlib
+    - hdf4
+    - hdf5
+
+  run:
+    - vs2015_runtime  # [win]
```
Comment (xhochy) [inline recipes/coda/meta.yaml:37]: This should be added automatically by thw compiler dependency.
```suggestion
```

File: recipes/coda/build.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+cmake --trace \
```
Comment (xhochy) [inline recipes/coda/build.sh:6]:
```suggestion
cmake $CMAKE_ARGS --trace \
```

File: recipes/coda/build.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  -DCODA_WITH_HDF5=True \
+  -DJPEG_INCLUDE_DIR="$PREFIX/include" \
+  -DJPEG_LIBRARY_DIR="${PREFIX}/lib" \
+  -DJPEG_LIBRARY="${PREFIX}/lib/libjpeg${SHLIB_EXT}" \
+  -DZLIB_INCLUDE_DIR="$PREFIX/include" \
+  -DZLIB_LIBRARY="${PREFIX}/lib/libz${SHLIB_EXT}" \
+  -DZLIB_LIBRARY_DIR="${PREFIX}/lib" \
+  ..
+
+make
```
Comment (xhochy) [inline recipes/coda/build.sh:21]:
```suggestion
make -j$CPU_COUNT
```

File: recipes/coda/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+{% set sha256 = "dd1cd0112d09cea29afa1568d644f54eccffdad2f5909708b519c1d7c647e9f1" %}
+{% set tag = version %}
+
+package:
+  name: coda
+  version: {{ version }}
+
+source:
+  url: https://github.com/stcorp/coda/releases/download/{{ tag }}/coda-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (xhochy) [inline recipes/coda/meta.yaml:11]: Please list the sha directly here

File: recipes/coda/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,60 @@
+{% set version = "2.22.1" %}
+{% set sha256 = "dd1cd0112d09cea29afa1568d644f54eccffdad2f5909708b519c1d7c647e9f1" %}
+{% set tag = version %}
```
Comment (xhochy) [inline recipes/coda/meta.yaml:3]: Not needed 
```suggestion
```

**Review by xhochy — COMMENTED** [review]
Suggestions apply to all recipes

File: recipes/coda/meta.yaml (hunk 5 above)
Comment (srepmub) [reply recipes/coda/meta.yaml]: thanks for the feedback! ^^ making changes..

**Pull request merged**